_utils/run.sh

```diff
@@ -0,0 +1,2 @@
+python slave_refresh.py
```

Comment:
I guess another alternative if this won't work is something like this and see if it runs in the same context:
```sh
$ python host_refresh.py && source .env
```